### ISSUE TITLE
Pasting from Google Docs in CKEditor 5 in Chrome makes everything bold

### ISSUE THREAD

**Issue by laryn** [backdrop/backdrop-issues#6472]
## Description of the bug

When pasting any text from Google Docs into a CKEditor 5 text editor in Chrome (doesn't seem to happen, at least not as consistently, in Firefox), the entire pasted section comes out in bold, even if none or only a portion of the copied text was bold in the Google Doc.

## Steps To Reproduce

To reproduce the behavior:
1. Open Chrome
2. Open [this sample Google Doc](https://docs.google.com/document/d/1pjLA5P0SKVtimJSiCgtMjHrmZWCB3TqqgpFdRoCS9gU/edit) in another tab and copy a section of it that is not completely bold
3. Spin up a demo Backdrop site
4. Create a basic page and paste the text you've copied above into the body

## Actual behavior

 - The entire text shows up as bold.

## Expected behavior

 - Only text that is bolded in the Google Doc should be bold.

## Additional information

 - The sample Google doc linked above [comes from this page](https://ckeditor.com/docs/ckeditor5/latest/features/pasting/paste-from-google-docs.html), which describes the "Paste from Office" plugin.
 - It looks like "Paste from Office" is included in Backdrop's CKE5 build: https://github.com/backdrop/backdrop/blob/7c611808d765a5401ca104268ce9b870a2063593/core/modules/ckeditor5/lib/scripts/concat-build.sh#L20
 - Is it possible it's not added to the default plugin list?

**Comment by laryn** [comment]
I took a bit of an educated stab in the dark to make a PR here and it seems to have worked!

![CleanShot 2024-04-24 at 11 49 54@2x](https://github.com/backdrop/backdrop-issues/assets/871421/bd6cae5a-8fc5-4018-8d68-889f82895a8f)

Ready for review, I think.

**Comment by klonos** [comment]
I have confirmed the bug. Not only is the text being pasted bolded, but trying to save the page results in a broken website/page. Here's what I've done:
- fresh Backdrop demo
- create a new page
- copy the first header and first paragraph of the [sample Google Doc](https://docs.google.com/document/d/1pjLA5P0SKVtimJSiCgtMjHrmZWCB3TqqgpFdRoCS9gU/edit)
- paste it into the body field -> all text is bolded 👎🏼 
- Add a title and try to save the page -> broken site 👎🏼  (only the admin bar rendered properly):
  ![image](https://github.com/backdrop/backdrop-issues/assets/2423362/2e39a76c-da5d-4a18-9f85-09e2d206eff1)

I then tested the same as above in the PR sandbox, and everything works as expected 👍🏼 ...code looks good as well, so RTBC.

Thanks @laryn
